cmd/aks-periscope/aks-periscope.go

```diff
@@ -40,45 +40,192 @@ func main() {
 		}
 	}
 
+	collectors, diagnosers, exporters := initializeComponents(creationTimeStamp, hostname)
+
+	//dataProducers includes all selected collectors and diagnosers
 	dataProducers := []interfaces.DataProducer{}
+	for _, c := range collectors {
+		dataProducers = append(dataProducers, c)
+	}
+	for _, d := range diagnosers {
+		dataProducers = append(dataProducers, d)
+	}
+
+	collectorGrp := new(sync.WaitGroup)
+	runCollectors(collectors, exporters, collectorGrp)
+	collectorGrp.Wait()
+
+	diagnoserGrp := new(sync.WaitGroup)
+	runDiagnosers(diagnosers, exporters, diagnoserGrp)
+	diagnoserGrp.Wait()
+
+	zipAndExportString, found := os.LookupEnv("ZIP_AND_EXPORT")
+	zipAndExport, parseErr := strconv.ParseBool(zipAndExportString)
+	if !found || parseErr != nil {
+		zipAndExport = true
+	}
+
+	if zipAndExport {
```

Comment:
The PR is about Kind cluster. Why all these changes in this file? What are we doing here?

Reply:
Let's have 3 PR:
- configuration of exporters + mechanism to run (or not) collectors / diagnosers
- PR for zip (not sure what we need to do here)
- PR to support kind cluster: if others PR are done, this one shall be very small

Reply:
ZipAndExport step shall not exist. We will remove it asap but I know other tools such as vscode are using it.
This zip step is a duplicate of the single exports per collector / diagnoser